examples/flipper/Cargo.toml

```diff
@@ -15,6 +15,12 @@ scale-info = { version = "2.6", default-features = false, features = ["derive"],
 [dev-dependencies]
 drink = { path = "../../drink" }
 
+# testing custom runtime
+frame-support = "28.0.0"
+frame-system = "28.0.0"
+pallet-contracts-mock-network = "3.0.0"
```

Comment:
actually, this flipper example didn't have any particular reason, but now, we can rename the whole example to e.g. `different-runtimes` and make it a showcase how to test a single contract against different environments